aioshelly/block_device/device.py

```diff
@@ -140,14 +144,17 @@ async def initialize(self) -> None:
                 await self.update_settings()
                 await self.update_status()
 
-                event_d: asyncio.Event = await self._coap_request("d")
-                # We need to wait for D to come in before we request S
-                # Or else we might miss the answer to D
-                await event_d.wait()
+                # Older devices has incompatible CoAP protocol (v1)
+                # Skip CoAP to avoid parsing errors
+                if self.firmware_supported:
+                    event_d: asyncio.Event = await self._coap_request("d")
```

Comment:
This typing looks redundant
```suggestion
                    event_d = await self._coap_request("d")
```